flutter-idea/src/io/flutter/editor/ActiveEditorsOutlineService.java

```diff
@@ -132,8 +132,7 @@ private void updateActiveEditors() {
       }
 
       for (final String path : obsoletePaths) {
-        final String filePathOrUri = getAnalysisServer().getAnalysisService().getLocalFileUri(path);
-        final FlutterOutlineListener listener = outlineListeners.remove(filePathOrUri);
+        final FlutterOutlineListener listener = outlineListeners.remove(path);
```

Comment:
As a follow up, reference the issue that is "fixed" with the line of code written like this, to allow more transparency to what is going on and the sensitive nature of this particular code.